taichi/runtime/llvm/runtime.cpp

```diff
@@ -268,6 +268,38 @@ f64 pow_f64(f64 a, f64 b) {
   return std::pow(a, b);
 }
 
+f32 pow_f32_i64(f32 x, i64 n) {
```

Comment:
Suggestion: pow_f32_i32?
We can clamp if rhs is i64 since pow result will overflow if rhs > INT_MAX anyway.

Reply:
Oh yes. (1 + 2^-23)^(2^31) = 1.5e+111 is not representable in f32.